src/Core/src/Hosting/Internal/AppHost.cs

```diff
@@ -6,21 +6,21 @@
 
 namespace Microsoft.Maui.Hosting.Internal
 {
-	internal class AppHost : IAppHost, IAsyncDisposable
+	class AppHost : IAppHost, IAsyncDisposable
```

Comment:
if it's not internal maybe shouldn't be on internal namespace ?

Reply:
Just removing the internal as it is the default.